### ISSUE TITLE
Inspect tests is an empty value when using the kitchen_ec2 driver

### ISSUE THREAD

**Issue by howdoicomputer** [test-kitchen/test-kitchen#1136]
## Description

When using the kitchen-ec2 driver, the inspec_tests attribute is an empty array. This means that tests are never transferred over to the spawned instance.

It works when you hard code the value like so:

```
    verifier:
      inspec_tests:
        - test/recipes
```

## Kitchen Version

```ruby
⋊> ~/w/w/c/tl_jenkins on CLOUDOPS-454 ⨯ kitchen --version                                                     13:27:34
Test Kitchen version 1.13.2
```

## ChefDK Version

If you are running test-kitchen via ChefDK, `chef --version` will provide additional relevent version details.

```ruby
⋊> ~/w/w/c/tl_jenkins on CLOUDOPS-454 ⨯ chef --version                                                        13:30:32
Chef Development Kit Version: 0.18.26
chef-client version: 12.14.77
delivery version: master (d86679335580be3de22996ef294b20d525889d8d)
berks version: 5.1.0
kitchen version: 1.13.2
```

## Ruby Version

If you are not using test-kitchen via ChefDK, please provide the output of ruby --version.

```ruby
⋊> ~/w/w/c/tl_jenkins on CLOUDOPS-454 ⨯ ruby -v                                                               13:30:42
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin16]
```

## Platform Version

macOS

## Replication Case

Using a kitchen.yml with these contents:

```
---
driver:
  name: ec2
  aws_ssh_key_id: 'jenkins-kitchen-test'
  instance_type: 'm3.large'
  security_group_ids: [ 'default', 'jenkins-test-kitchen' ]
  tags: { 'Role': 'test-kitchen', 'Cookbook': 'tl_jenkins' }

verifier:
  name: inspec

transport:
  ssh_key: ~/.ssh/jenkins-kitchen-test

provisioner:
  name: chef_zero

platforms:
  - name: ubuntu-16.04

suites:
  - name: default
    run_list:
      - recipe[tl_jenkins::default]
    attributes:
```

Run `kitchen verify -l=debug`.

## Kitchen Output

```ruby
⋊> ~/w/w/c/tl_jenkins on CLOUDOPS-454 ⨯ env KITCHEN_YAML=".kitchen.ec2.yml" kitchen verify -l=debug           13:31:26
-----> Starting Kitchen (v1.13.2)
D      Berksfile found at /Users/thampton/workspace/work/cookbooks/tl_jenkins/Berksfile, loading Berkshelf
D      Berkshelf 5.1.0 library loaded
-----> Verifying <default-ubuntu-1604>...
D      Initialize InSpec
       Use `/Users/thampton/workspace/work/cookbooks/tl_jenkins/test/recipes/default` for testing
D      Running tests from: []

Summary: 0 successful, 0 failures, 0 skipped
       Finished verifying <default-ubuntu-1604> (0m2.24s).
-----> Kitchen is finished. (0m7.51s)
```

## Kitchen Diagnose

```
    verifier:
      chef_omnibus_root: "/opt/chef"
      command_prefix:
      ftp_proxy:
      http_proxy:
      https_proxy:
      inspec_tests: []
      kitchen_root: "/Users/thampton/workspace/work/cookbooks/tl_jenkins"
      log_level: :info
      name: inspec
      root_path: "/tmp/verifier"
      sudo: true
      sudo_command: sudo -E
      suite_name: default
      test_base_path: "/Users/thampton/workspace/work/cookbooks/tl_jenkins/test/recipes"
```

**Comment by coderanger** [comment]
As I said on the other ticket, I don't know what you are asking for. `kitchen-inspec` doesn't work by transferring things to the test instance, so there is nothing to transfer. If you are trying to use busser with inspec, I think you'll have to write a plugin for that yourself.

**Comment by howdoicomputer** [comment]
@coderanger Maybe I'm not phrasing it right? Would 'run against' be better phrasing than 'transfer to'? I'm not really trying to use busser - just inspec.

I wanted these tests:

```

describe port(80) do
  it { should be_listening }
end

describe runit_service('jenkins') do
  it { should be_running }
end
```

...to be ran against the EC2 instance that kitchen-ec2 creates. When I use kitchen-vagrant, the path to my tests is able to be picked up but when I use kitchen-ec2 then the path isn't able to be resolved.

I was able to get what I wanted after hard coding the path to my tests - which I didn't need to do with the vagrant driver. Once I did that, `kitchen verify` was able to run inspec tests.
